react/rules/BEL.js

```diff
@@ -0,0 +1,61 @@
+import initialize from './initializeCountryPhone'
+
+export default {
+  country: 'BEL',
+  personalFields: [
+    {
+      name: 'firstName',
+      maxLength: 100,
+      label: 'firstName',
+      required: true,
+    },
+    {
+      name: 'lastName',
+      maxLength: 100,
+      label: 'lastName',
+      required: true,
+    },
+    {
+      name: 'email',
+      maxLength: 100,
+      label: 'email',
+      hidden: true,
+    },
+    {
+      name: 'homePhone',
+      maxLength: 30,
+      label: 'homePhone',
+      ...getPhoneFields(phoneCountryCode),
```

Comment:
This will throw an error since you removed the functions...

Reply:
Call to Function removed.